### ISSUE TITLE
Doesn't work. Errors out when I use the Python web UI and click "Start Training" , with "cannot import 'packaging' form 'pkg_resources'

### ISSUE THREAD

**Issue by nikolatesla20** [ostris/ai-toolkit#228]
## This is for bugs only

Did you already ask [in the discord](https://discord.gg/VXmU2f5WEU)?

No

You verified that this is a bug and not a feature request or question by asking [in the discord](https://discord.gg/VXmU2f5WEU)?

Yes

## Describe the bug

When I use the Python web UI and set everything up, the click Start Training, I get a python error 
![image](https://github.com/user-attachments/assets/c2a865b0-5afd-4d9c-b9aa-6e55f900128e)

**Comment by cluiverto** [comment]
pip install setuptools==69.5.1

**Comment by PsyChip** [comment]
yes, the exact version solved the issue. now fiddling with config/examples/generate.example.yaml to run a test interference but no luck. any ideas?

**Comment by cluiverto** [comment]
what error?

**Comment by PsyChip** [comment]
> what error?

actually there is no error, looking for a shortcut to run interference after training

**Comment by hasselburglar** [comment]
> pip install setuptools==69.5.1

This did not work for me - still getting the same error even after upgrading setuptools to the latest version after trying your suggestion first.
